core/src/com/unciv/logic/automation/Automation.kt

```diff
@@ -85,15 +86,23 @@ object Automation {
         } else {
             if (city.civ.gold < 0 && city.civ.stats.statsForNextTurn.gold <= 0)
                 yieldStats.gold *= 2 // We have a global problem
+            yieldStats.gold * city.civ.getPersonality().scaledFocus(PersonalityValue.Gold)
```

Comment:
`*=`, I assume, also for the rest of this function

Reply:
Most of the cost of city.civ.getPersonality is looking up the ruleset... but, hmm, yeah, probably should be a val

Haven't considered worker automation (I mostly looked at where the existing victory focus code was, none is there to my knowledge. Would rather work on it as a separate PR